tests/monster_test_generated.rs

```diff
@@ -638,7 +634,7 @@ impl<'b> flatbuffers::Push for &'b Vec3 {
 
 
 impl Vec3 {
-  pub fn new<'a>(_x: f32, _y: f32, _z: f32, _test1: f64, _test2: Color, _test3: &'a Test) -> Self {
```

Comment:
@CasperN Is there a conceivable issue with having two references in the args?

Before, it would look like:

```rust
pub fn new<'a>(_foo: &'a Foo, _bar: &'a Bar) -> Self { ... }
```

But with this change it looks like it will desugar to:

```rust
pub fn new<'a, 'b>(_foo: &'a Foo, _bar: &'b Bar) -> Self { ... }
```

Reply:
There shouldn't be an issue: Flatbuffers-structs don't hold references and aren't parameterized by any lifetime. We don't need to indicate how the lifetime of `new`'s output relates to the lifetime of its input because it doesn't.

Reply:
Also, this worked fine

https://gist.github.com/CasperN/2dda56bd3f7e62208d774cea16072fcd